docs/04-how-we-work/continuous-learning.md

```diff
@@ -0,0 +1,65 @@
+reach it's fullest potential in the world.
+
+Many organizations operate in silos, as the following image represents:
+
+![Organizational Silos](https://cdn-images-1.medium.com/max/1600/1*Mi-WprLxdBcreUjHLWLtBg.png)
+
+Silos often create challenges around openness and discourage broader participation from all members of the organization.
+ At CivicActions, openness is in our value system and we believe every member of our team should be able to participate in 
+ most aspects of the organization. We aim to offer tools to help improve our organization and our projects based on 
+ what we learn.
+ 
+Agile teams (projects, proposals, labs, prodev, etc) generate improvements through actual work. We desire orthogonal 
+structures that harness organizational feedback from all of the Agile teams. This concept is represented 
+by the following visual:
+
+![Agile Guilds and Agile Teams](https://cdn-images-1.medium.com/max/1600/1*Nk9G_KFq6gMGOLMxZTM3rQ.png)
+
+Such a structure allows for any Agile team to have a framework for sharing their learning back with the 
+organization.
+ 
+### Project Learning
+
+Projects are defined in broad context to capture any sort of work performed within the team. This could be a 
+development project for a client, writing a sales proposal, a marketing initiative, or even professional development
+activities. 
+
+### Organizational Learning
+
+As we work, team members may identify considerations or improvements outside of projects that generally affect how 
+we work. Team members may organically come up with ideas that they want to share with CivicActions. We aim to provide 
+a practice for harnessing and discussing ideas within the organization. We explored frameworks like Holacracy 
+and Agile Guilds, which provide interfaces for team members to share thoughts with the broader organization. These 
+concepts motivated our implementation, which is described in the practices section of this page.
+
+## Practices
+
+This section articulates our practices for continuous learning within our organization.
+
+## Improvements
+Improvements represent opportunities that stem from the work we do to facilitate discussion and next steps toward 
+organizational clarity. In our Agile process, improvements most directly relates to ideas or feedback that come from 
+retrospectives. Holacracy uses the term "tensions" for this concept. We have chosen "improvements".
+
+### Retrospectives
+
+Our Scrum teams hold retrospectives to identify both organizational and project improvements based on 
+the work performed during projects. Projects can iteratively make changes that improve efficiency, operations, 
+and much more. This approach encourages team members to learn, grow, and experiment in ways to more effectively serve 
+our customers without fear of failure. More information is provided on the 
+[retrospectives](agile-baseline/02-process/practices/sprint-retrospectives.md) page.
+
+### Governance Meetings and Agile Guilds
```

Comment:
We should look to firm up the naming of Agile Guilds with whatever we land on above (possibly Practice Areas)